### DISCUSSION TITLE
Referensi Lanjutan untuk Jenis Sebaran?

### DISCUSSION THREAD

**Discussion by taruma** [hidrokit/hidrokit#157] [Tanya Jawab] [answered]
Saya ingin mencari referensi utama untuk tabel berikut ini:

![image](https://user-images.githubusercontent.com/1007910/155874822-baa515bf-761d-447e-bd5a-fe3cc570312e.png)

![image](https://user-images.githubusercontent.com/1007910/155874925-e837a93b-97e6-4da9-b7e4-3f50291b2ba1.png)

Saya sudah coba buka dari Soewarno[^1], dan saya kesulitan untuk menunjukkan asal usul tabel diatas. Mungkin terdapat di jilid kedua (saya hanya memiliki jilid pertamanya saja). Terkadang ada perbedaan (seperti contoh diatas untuk log normal), sehingga cukup perlu tahu referensi yang lebih tepat. 

[^1]: Soewarno. (1995). hidrologi: Aplikasi Metode Statistik untuk Analisa Data. NOVA.

**Comment by taruma** [comment, marked as answer]
Pada SNI: 2415:2016 Tata Cara Perhitungan Debit Banjir Rencana disinggung terkait jenis khas masing-masing distribusi:

![image](https://user-images.githubusercontent.com/1007910/156682799-e0b3f5ac-d31a-4900-afaa-687b821723ed.png)
![image](https://user-images.githubusercontent.com/1007910/156682827-48bf8238-0012-4b46-9513-cb3442e0f173.png)

Jadi jika mengikuti pernyataan pada SNI dapat dituliskan sebagai berikut:
Distribusi | Syarat | Catatan
:-: | --------- | ---------
Normal | C_s ~ 0 <br> C_k ~ 3 | -
Log Normal | C_s ~ 3 <br> C_s > 0 <br> C_s ~ 3C_v | -
Gumbel Tipe I | C_s ~ 1.1396 <br> C_k ~ 5.4002 | -
Log Pearson Tipe III | - | tidak memiliki sifat khas

**Reply by rudianto551** [reply]
bisa dijelaskan lagi

**Reply by taruma** [reply]
bagian mana yang perlu dijelaskan lagi? 🤔
